src/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsLowTrust.cs

```diff
@@ -0,0 +1,178 @@
+﻿using System.Diagnostics.Tracing;
+using Xunit;
+using System;
+using System.Linq;
+using System.Security;
+using System.Security.Permissions;
+using System.Security.Policy;
+
+namespace BasicEventSourceTests
+{
+    
+    public class TestsLowTrust
+    {
+        /// <summary>
+        /// Very simple low trust test that calls directly into a low trust method
+        /// without creating a separate app domain
+        /// </summary>
+        [Fact]
+        public void Test_EventSource_LowTrust()
+        {
+            TestUtilities.CheckNoEventSourcesRunning("Start");
+            Console.WriteLine("Running Array.CreateInstance() in low trust...");
+            RunLowTrustTest();
+            Console.WriteLine("Success...");
+            TestUtilities.CheckNoEventSourcesRunning("Stop");
+        }
+
+        [SecurityPermission(SecurityAction.PermitOnly, Execution = true)]
```

Comment:
Is this going to have any effect on CoreCLR with the hosts we use for running tests?